generators/server/templates/settings.gradle.ejs

```diff
@@ -26,6 +26,16 @@ pluginManagement {
 <%_ } _%>
         // jhipster-needle-gradle-plugin-management-repositories - JHipster will add additional entries here
     }
+<%_ if (databaseMigrationLiquibase) { _%>
+    buildscript {
+        repositories {
+            mavenCentral()
+        }
+        dependencies {
+            classpath group: 'org.liquibase', name: 'liquibase-core', version: "${liquibaseCoreVersion}"
```

Comment:
Does it work by using dependencyManagement imported version like?
https://github.com/jhipster/generator-jhipster/blob/9b50af5c99b44792e2b5b087b725e208c60d3c29/generators/liquibase/templates/gradle/liquibase.gradle.ejs#L36